test/PLCRVotingChallenge.test.js

```diff
@@ -0,0 +1,271 @@
+import lkTestHelpers from 'lk-test-helpers'
+const { expectRevert, expectEvent, expectThrow } = lkTestHelpers(web3)
```

Comment:
this won't be compatible with the latest lk-test-helpers .. but I can probably just rebase the changes after this gets merged